src/components/Table/Table.qa.stories.tsx

```diff
@@ -378,3 +388,188 @@ const flatRows: GridDataRow<BeamRow>[] = [
     },
   })),
 ];
+
+/* Input Field Story and data */
+type InputFieldRows = SimpleHeaderAndData<AuthorInput>;
+
+export function InputFieldStates() {
+  const { getFormState } = useFormStates<AuthorInput>({
+    config: formConfig,
+    getId: (o) => o.id!,
+  });
+
+  return (
+    <>
+      <h2 css={Css.mb1.$}>Fixed Table Styles</h2>
+      <GridTable<InputFieldRows>
+        style={beamFixedStyle}
+        columns={inputFieldColumns(getFormState)}
+        rows={inputFieldRows}
+        stickyHeader
+      />
+
+      <h2 css={Css.mt5.mb1.$}>Flexible Table Styles</h2>
+      <GridTable<InputFieldRows>
+        style={beamFlexibleStyle}
+        columns={inputFieldColumns(getFormState, true)}
+        rows={inputFieldRows}
+        stickyHeader
+      />
+    </>
+  );
+}
+
+function inputFieldColumns(getFormState: (author: AuthorInput) => ObjectState<AuthorInput>, flexible?: boolean) {
+  function applyStateProps(id: string) {
+    return {
+      internalProps: { forceFocus: id === "a:3", forceHover: id === "a:2" },
+      errorMsg: id === "a:4" ? "Description of error" : undefined,
+    };
+  }
+  return [
+    column<InputFieldRows>({
+      header: "Name",
+      data: (data, row) => ({
+        content: () => {
+          const os = getFormState(data);
+          return <BoundTextField field={os.firstName} {...applyStateProps(row.id)} />;
+        },
+      }),
+    }),
+    column<InputFieldRows>({
+      header: "Biography",
+      data: (data, row) => ({
+        content: () => {
+          const os = getFormState(data);
+          return flexible ? (
+            <BoundTextAreaField field={os.bio} preventNewLines {...applyStateProps(row.id)} />
```

Comment:
Can you think of any cute ways to infer `preventNewLines` from `PresentationContext`? Np if not...

Reply:
We do have a `PresentationSettings.wrap` property that we use, but I don't think thats really all too clear. I'm not sure exactly when else we'd want `preventNewLines` to be applied. Right now it only happens in schedulesV2. It's for when we really just want a text field, but also want the value to wrap. Hmm... was thinking maybe if a user has a `TextField` component within a presentation setting of `wrap: true`, then we'd instead display the `TextAreaField` with `preventNewLines` enabled. But I don't think that is super clear. Not sure if the `wrap` setting is all that flushed out yet and don't really want to pile things on it at the moment. I think the only consumer of `wrap` is the Chips component.

Reply:
Agreed that maybe someday `wrap` would drive this if we end up doing it a lot, but good to hold off for now.